integration_tests/mockApis/sendLegalMail.ts

```diff
@@ -192,9 +192,10 @@ const stubVerifyDuplicateBarcode = (): SuperAgentRequest =>
         status: 400,
         errorCode: {
           code: 'DUPLICATE',
-          userMessage: 'Someone scanned this barcode at 9:11 am on 8 December 2021 at LEI. It may be an illegal copy.',
+          userMessage:
+            'Someone scanned this barcode at 9:11 am on 8 December 2021 at HMP Altcourse. It may be an illegal copy.',
           scannedDate: '2021-12-08T09:11:23Z',
-          scannedLocation: 'LEI',
+          scannedLocation: 'ACI',
```

Comment:
Set our stub response to be ACI because our stub prison data only contains a few prisons, with LEI not being one of them